compiler/src/dotty/tools/dotc/typer/Inliner.scala

```diff
@@ -439,12 +444,18 @@ class Inliner(call: tpd.Tree, rhsToInline: tpd.Tree)(implicit ctx: Context) {
             case _ => tree
           }
         case tree: Ident =>
+          def span =
+            if (tree.name == nme.WILDCARD) tree.span // From type match
+            else if (tree.symbol.isTypeParam && tree.symbol.owner.isClass) tree.span // TODO is this the correct span?
+            else paramSpan(tree.name)
```

Comment:
Maybe add one line document to `def span`?

Reply:
Added a comment and renamed `span` to `argSpan` to make it clearer.